src/nvim/context.h

```diff
@@ -1,7 +1,5 @@
 #pragma once
 
-#include <stddef.h>
```

Comment:
This is needed by the generated header

Reply:
Thanks :) If IWYU doesn't see that, and the build passes... hmm...